spec/models/note_spec.rb

```diff
@@ -414,7 +414,7 @@
   end
 
   describe 'lang_from_cloud' do
-    Setting['advanced.detect_language_sample_length'] = 100
+    ENV['detect_language_sample_length'] = '100'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.